src/novi/basics/ElectricPokemon.java

```diff
@@ -12,6 +12,26 @@ public ElectricPokemon(String name, int level, int hp, String sound, String food
         this.emotion = emotion;
     }
 
+    // gettersNsetters -----------------------------------------------------------------------------
```

Comment:
Ik mis nog een .gitignore in je project waarin je je Assets en out map mag zetten